pkg/services/sqlstore/annotation.go

```diff
@@ -42,7 +42,7 @@ func NewSQLAnnotationRepo(sql *SQLStore) SQLAnnotationRepo {
 }
 
 func (r *SQLAnnotationRepo) Save(item *annotations.Item) error {
-	return inTransaction(func(sess *DBSession) error {
+	return r.sql.WithTransactionalDbSession(context.Background(), func(sess *DBSession) error {
```

Comment:
Nit. Would be nice with context as first argument to the method. Otherwise would suggest replacing `context.Background()` with `context.TODO()` and fix context in a separate PR. Up to you